code/client/managedsoftwareupdate

```diff
@@ -279,6 +279,17 @@ def doRestart():
     if not consoleuser or consoleuser == u'loginwindow':
         # no-one is logged in or we're at the loginwindow
         time.sleep(5)
+        #check to see if we should perform an AuthRestart
```

Comment:
Don't you also need to check to see if FileVault is enabled? What happens if moron admin specifies these keys, yet FileVault is not enabled/active on the boot disk?